EMS/core-bundle/src/Commands.php

```diff
@@ -65,6 +65,8 @@ final class Commands
 
     public const string XLIFF_EXTRACT = 'emsco:xliff:extract';
     public const string XLIFF_UPDATE = 'emsco:xliff:update';
+    public const string USER_ADD_GROUP = 'emsco:user:add-group';
+    public const string USER_DELETE_GROUP = 'emsco:user:delete-group';
```

Comment:
```suggestion
    public const string USER_DELETE_GROUP = 'emsco:user:remove-group';
```

We aren't deleting a group